### PR TITLE
Release notes for Percona Server 5.6.38-83.0 (5.6)

### PR DESCRIPTION
**Pull request by hrvojem**
Fixed broken upstream-bug-fixes table

### CODE REVIEW

File: doc/source/performance/aio_page_requests.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+=======================================
+Multiple page asynchronous I/O requests
+=======================================
+
+I/O unit size in |InnoDB| is only one page, even if doing read ahead. 16KB
+I/O unit size is too small for sequential reads, and much less efficient than
+larger I/O unit size.
+
+|InnoDB| uses Linux Native I/O (``aio``) by default. By submitting multiple
```
Comment (laurynas-biveinis) [inline doc/source/performance/aio_page_requests.rst]: InnoDB uses Linux asynchronous I/O

File: doc/source/performance/aio_page_requests.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+consecutive 16KB read requests at once, Linux internally can merge requests and
+reads can be done more efficiently.
+
+On modern HDD RAID 1+0 environment, more than 1000MB/s disk reads can be
```
Comment (laurynas-biveinis) [inline doc/source/performance/aio_page_requests.rst]: Where is this taken from? Please attach attribution to this claim. How old is "modern" here?

File: doc/source/performance/aio_page_requests.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+achieved by submitting 64 consecutive pages requests at once, while only
+160MB/s disk reads is shown by submitting single page request.
+
+With this feature |InnoDB| submits multiple page I/O requests.
+
+Version Specific Information
+============================
+
+ * :rn:`5.6.38-83.0`- Feature ported from a *Facebook MySQL*  patch.
```
Comment (laurynas-biveinis) [inline doc/source/performance/aio_page_requests.rst]: s/a/the

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+ requires all the key columns to come before any ``BLOB`` columns. This
+ requirement however was not enforced, allowing creating MEMORY tables in
+ unsupported column configurations, which then crashed or lose data in usage.
+ Bug fixed :bug:`1731483`.
+
+ If an I/O syscall returned an error during the server shutdown with
+ :ref:`threadpool` enabled, a mutex could be left locked. Bug fixed
+ :bug:`1702330` (*Daniel Black*).
+
+ After fixing bug :bug:`1668602`, bug :bug:`1539504`, and bug :bug:`1313901`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.38-83.0.rst]: Please re-check the bug, this is wrong description

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+ temporary tables were not logged in the binary log if under ``ROW`` and
+ ``MIXED`` based binary log mode. Bug fixed :bug:`1711781`.
+
+ Under concurrent environment running ``ANALYZE TABLE`` could lead to a
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.38-83.0.rst]: ANALYZE TABLE and a long running query on the same table, then the new queries get the WfTF state.

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+ :ref:`threadpool` enabled, a mutex could be left locked. Bug fixed
+ :bug:`1702330` (*Daniel Black*).
+
+ After fixing bug :bug:`1668602`, bug :bug:`1539504`, and bug :bug:`1313901`,
+ temporary tables were not logged in the binary log if under ``ROW`` and
+ ``MIXED`` based binary log mode. Bug fixed :bug:`1711781`.
+
+ Under concurrent environment running ``ANALYZE TABLE`` could lead to a 
+ situation when hundreds of tables were in a ``Waiting for table flush`` state.
+ Fixed by releasing lock on table statistics after query plan is created and
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.38-83.0.rst]: Remove the "releasing lock" part

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+ :bug:`1702330` (*Daniel Black*).
+
+ After fixing bug :bug:`1668602`, bug :bug:`1539504`, and bug :bug:`1313901`,
+ temporary tables were not logged in the binary log if under ``ROW`` and
+ ``MIXED`` based binary log mode. Bug fixed :bug:`1711781`.
+
+ Under concurrent environment running ``ANALYZE TABLE`` could lead to a 
+ situation when hundreds of tables were in a ``Waiting for table flush`` state.
+ Fixed by releasing lock on table statistics after query plan is created and 
+ by stopping ``ANALYZE TABLE`` flushing affected tables from the table
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.38-83.0.rst]: flushing affected InnoDB and TokuDB (in 5.7 MyRocks too)

File: doc/source/upstream-bug-fixes.rst [hunk 8]
```diff
@@ -4,22 +4,28 @@
  List of upstream |MySQL| bugs fixed in |Percona Server| 5.6
 =============================================================
 
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`82967`                                                                             |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: No title?

File: doc/source/upstream-bug-fixes.rst [hunk 9]
```diff
@@ -4,22 +4,28 @@
...
+|:Upstream bug: :mysqlbug:`82967`                                                                             |  
+|:Launchpad bug: :bug:`1622992`                                                                               |
+|:Upstream state: N/A                                                                                         |
+|:Fix Released: :rn:`5.6.37-82.2`                                                                             |
+|:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83814` - Add support for OpenSSL 1.1                                               |
 |:Launchpad bug: :bug:`1702903`                                                                               |
-|:Upstream state: Verified (checked on 2017-07-28)                                                            |
+|:Upstream state: Verified (checked on 2017-08-25)                                                            |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: 2017-12-08 ?

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+ If an I/O syscall returned an error during the server shutdown with
+ :ref:`threadpool` enabled, a mutex could be left locked. Bug fixed
+ :bug:`1702330` (*Daniel Black*).
+
+ After fixing bug :bug:`1668602`, bug :bug:`1539504`, and bug :bug:`1313901`,
+ ``CREATE/DROP TEMPORARY TABLE`` statements were forbidden incorrectly in
+ transactional contexts, including function and trigger calls, even when
+ they required no binary logging at all. Bug fixed :bug:`1711781`.
+
+ Under concurrent environment running ``ANALYZE TABLE`` could lead to a
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.38-83.0.rst]: running ANALYZE TABLE while a long-running query is accessing the same table in parallel could lead to a situation where new queries on the same table are blocked in a WFTF state.

Also did not address my previous review comment on storage engines of those tables

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Review by laurynas-biveinis — APPROVED** [review]
TokuDB needs @georgelorchpercona review

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+ |Percona Server| has implemented :ref:`innodb_fragmentation_count`.
+
+ |Percona Server| has implemented support for :ref:`aio_page_requests`
+ This feature was ported from a *Facebook MySQL* patch.
+
+ |Percona Server| has implemented |TokuDB| :ref:`integration
+ <tokudb_performance_schema>` with ``PERFORMANCE_SCHEMA``.
+
+ |Percona Server| packages are now available for *Ubuntu 17.10 (Artful)*.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.38-83.0.rst:31]: Are we not skipping artful for now?

**Review by george-lorch — APPROVED** [review]
Other than artful question, LGTM

File: doc/source/release-notes/Percona-Server-5.6.38-83.0.rst (hunk 11 above)
Comment (hrvojem) [reply doc/source/release-notes/Percona-Server-5.6.38-83.0.rst:31]: It was fixed this morning.

**Pull request merged**